Tests.CSharp/Homework6/Tests.cs

```diff
@@ -83,7 +83,7 @@ public BasicTests(CustomWebApplicationFactory<App.Startup> factory)
     [Homework(Homeworks.HomeWork6)]
     public async Task TestParserDividingByZero()
     {
-        await RunTest("15.6", "0", "Divide", "DivideByZero", HttpStatusCode.OK, true);
+        await RunTest("15.6", "0", "Divide", "DivideByZero", HttpStatusCode.BadRequest, true);
```

Comment:
Не правильно меня понял. Конкретно калькулятор не должен делить на 0 (не делить вообще или возвращать Error). Но давай придерживаться исходных тестов, считай, что это требования проекта. 
```suggestion
        await RunTest("15.6", "0", "Divide", "DivideByZero", HttpStatusCode.Ok, true);
```